metricbeat/module/elasticsearch/node_stats/data.go

```diff
@@ -83,7 +86,50 @@ var (
 				"count":   c.Int("count"),
 				"deleted": c.Int("deleted"),
 			}),
+			"get": c.Dict("get", s.Schema{
+				"time": s.Object{
+					"ms": c.Int("time_in_millis"),
+				},
+				"total": s.Object{
+					"count": c.Int("total"),
+				},
+			}),
+			"merges": c.Dict("merges", s.Schema{
+				"total_time": s.Object{
+					"ms": c.Int("total_time_in_millis"),
+				},
+				"total": s.Object{
+					"count": c.Int("total"),
+				},
+			}),
+			"refresh": c.Dict("refresh", s.Schema{
+				"total_time": s.Object{
+					"ms": c.Int("total_time_in_millis"),
+				},
+				"total": s.Object{
+					"count": c.Int("total"),
+				},
+			}),
```

Comment:
We could define a shared schema and reuse it, like `threadPoolStatsSchema` below, instead of redefining it. This would help to avoid unnecessary duplications and highlight intentional differences (like with `"get"`).

```go
poolTotalsSchema := s.Schema{
  "total_time": s.Object{
    "ms": c.Int("time_in_millis"),
  },
  "total": s.Object{
    "count": c.Int("total"),
  },
}

// ...
"merges": c.Dict("merges", poolTotalsSchema),
"refresh": c.Dict("refresh", poolTotalsSchema),
```